core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2703,10 +2695,10 @@ struct ViewDataHandle<
                                    Kokkos::CudaUVMSpace>::value))
 #endif
                 && (!Traits::memory_traits::is_atomic))>::type> {
-  typedef typename Traits::value_type value_type;
-  typedef typename Traits::value_type* KOKKOS_RESTRICT handle_type;
-  typedef typename Traits::value_type& KOKKOS_RESTRICT return_type;
```

Comment:
Ouch

Reply:
I don't see any of these in the `develop` branch, cf https://github.com/kokkos/kokkos/blob/develop/core/src/impl/Kokkos_ViewMapping.hpp.